release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -123,6 +139,24 @@ console. This lets you view metrics that bring transparency to the
 [id="ocp-4-4-web-console"]
 === Web console
 
+[id="ocp-4-4-marketplace-integration-in-operator-hub"]
+==== IBM Marketplace integration in OperatorHub
+
+IBM Marketplace is now integrated in the OperatorHub, which is located in the
+{product-title} web console.
+
+[id="ocp-4-4-edit-apps-in-topology-view"]
+==== Edit applications in the Topology view
+
+You can now edit applications from the Developer perspective by using the
+*Topology* view.
```

Comment:
Is this talking specifically about a UI control or the view itself? According to the repo guidelines, only bold the former.

Reply:
Only the view itself. I'll update both of these cases.